runtime/core/src/main/java/io/atlasmap/converters/DateConverter.java

```diff
@@ -139,32 +135,104 @@ public Short toShort(Date value) throws AtlasConversionException {
     }
 
     @AtlasConversionInfo(sourceType = FieldType.DATE_TIME, targetType = FieldType.DATE)
-    public java.sql.Date toSqlDate(Date date, String sourceFormat, String targetFormat)
-            throws AtlasConversionException {
+    public java.sql.Date toSqlDate(Date date, String sourceFormat, String targetFormat) {
         return DateTimeHelper.convertDateToSqlDate(date, sourceFormat);
     }
 
     @AtlasConversionInfo(sourceType = FieldType.DATE_TIME, targetType = FieldType.TIME)
-    public Time toSqlTime(Date date, String sourceFormat, String targetFormat) throws AtlasConversionException {
+    public Time toSqlTime(Date date, String sourceFormat, String targetFormat) {
         return DateTimeHelper.convertDateToTime(date, targetFormat);
     }
 
     @AtlasConversionInfo(sourceType = FieldType.DATE_TIME, targetType = FieldType.STRING)
-    public String toString(Date date) throws AtlasConversionException {
+    public String toString(Date date) {
         // by default Instant.toString returns an ISO-8601 representation of the instant
         return date.toInstant().toString();
 
     }
 
     @AtlasConversionInfo(sourceType = FieldType.DATE_TIME, targetType = FieldType.DATE_TIME)
-    public Timestamp toSqlTimestamp(Date date) throws AtlasConversionException {
+    public Timestamp toSqlTimestamp(Date date) {
         return Timestamp.from(date.toInstant());
     }
 
+    @AtlasConversionInfo(sourceType = FieldType.DATE_TIME_TZ, targetType = FieldType.DATE_TIME_TZ)
+    public ZonedDateTime toZonedDateTime(Calendar calendar) {
+        return DateTimeHelper.toZonedDateTime(calendar);
+    }
+
+    @AtlasConversionInfo(sourceType = FieldType.DATE_TIME, targetType = FieldType.DATE_TIME_TZ)
+    public ZonedDateTime toZonedDateTime(Date date, String sourceFormat, String targetFormat) {
+        return DateTimeHelper.toZonedDateTime(date, sourceFormat == null ? targetFormat : sourceFormat);
+    }
+
+    @AtlasConversionInfo(sourceType = FieldType.DATE, targetType = FieldType.DATE_TIME_TZ)
+    public ZonedDateTime toZonedDateTime(LocalDate date, String sourceFormat, String targetFormat) {
```

Comment:
To the LocalDateConverter, ditto for others doesn't have `java.util.Date` as a parameter.